### ISSUE TITLE
amplify_authenticator-0.1.0-rc.6. Error: The getter 'onInverseSurface' isn't defined for the class 'ColorScheme'

### ISSUE THREAD

**Issue by jpablogn** [aws-amplify/amplify-flutter#1381]
Trying to do this:
https://docs.amplify.aws/lib/auth/getting-started/q/platform/flutter/#authentication-with-amplify

I use this code:
https://pub.dev/packages/amplify_authenticator

But it shows this error:
```
/[C:/_Programas/flutter/.pub-cache/hosted/pub.dartlang.org/amplify_authenticator-0.1.0-rc.6/lib/src/widgets/authenticator_banner.dart:50:25](): Error: The getter 'onInverseSurface' isn't defined for the class 'ColorScheme'.
- 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/[C:/_Programas/flutter/packages/flutter/lib/src/material/color_scheme.dart]()').
package:flutter/…/material/color_scheme.dart:1
Try correcting the name to the name of an existing getter, or defining a getter or field named 'onInverseSurface'.
      theme.colorScheme.onInverseSurface;
                        ^^^^^^^^^^^^^^^^
3

FAILURE: Build failed with an exception.

* Where:
Script '[C:\_Programas\flutter\packages\flutter_tools\gradle\flutter.gradle]()' line: 1070

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command '[C:\_Programas\flutter\bin\flutter.bat]()'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at [https://help.gradle.org]()

BUILD FAILED in 34s
Exception: Gradle task assembleDebug failed with exit code 1
```
In this file:
\flutter\.pub-cache\hosted\pub.dartlang.org\amplify_authenticator-0.1.0-rc.6\lib\src\widgets\authenticator_banner.dart

I do this change to make it work:
  /*
  final foregroundColor = theme.snackBarTheme.contentTextStyle?.color ?? theme.colorScheme.onInverseSurface;
  */
  final foregroundColor = Colors.orange;

**Comment by Jordan-Nelson** [comment]
Hi @jpablogn - What version of flutter are you using?

**Comment by Jordan-Nelson** [comment]
I can reproduce this with flutter 2.8.1. 

I believe `onInverseSurface` was added in flutter 2.10.0. I will open a PR to remove the use of it so that using flutter 2.10.0+ isn't required. Thanks for bringing this to our attention.

**Comment by jpablogn** [comment]
>> flutter --version
Flutter 2.8.1 • channel stable • https://github.com/flutter/flutter.git   
Framework • revision 77d935af4d (10 weeks ago) • 2021-12-16 08:37:33 -0800
Engine • revision 890a5fca2e
Tools • Dart 2.15.1

Thank you.

**Comment by Jordan-Nelson** [comment]
The fix for this was merged. It is just pending a release now. I will update this issue after the next release.

**Comment by Jordan-Nelson** [comment]
This has been addressed in v0.1.0.

Also just wanted to callout that the docs for the authenticator have moved. You can find them here: https://ui.docs.amplify.aws/components/authenticator?platform=flutter

I am going to close this issue out.
